README.md

```diff
@@ -1,81 +1,65 @@
+VIP Real-Time Collaboration (VIP RTC) enables multiple users to edit the same WordPress content simultaneously, using the Block Editor. Team members can collaborate on Posts, Pages, and Custom Post Types with near-instant updates, seeing each other's changes as they type and configure content.
 
 ## Table of Contents
 
 - [Features](#features)
-- [Supported Content Types](#supported-content-types)
+- [Supported content types](#supported-content-types)
 - [Requirements](#requirements)
-- [Installation](#installation)
 - [FAQ](#faq)
 - [Development](#development)
-- [Support](#support)
 - [Credits](#credits)
 
 ## Features
 
 - **Real-Time Collaboration**: Multiple users can edit content simultaneously with instant updates
 - **Smart Conflict Resolution**: Automatically merges changes to help prevent overwriting each other's work
 - **Visual Collaboration Indicators**: See live cursors, user avatars, and editing activity from team members
-- **Seamless WordPress Integration**: Uses existing WordPress permissions - no additional setup required
 - **Reliable Connection Management**: Graceful handling of network interruptions with automatic reconnection
 
-## Supported Content Types
+## Supported content types
 
-The plugin works with:
+Out of the box, the plugin works with standard WordPress posts and pages. Custom Post Types can opt-in by adding support for `collaborative-editing`:
 
-- **Posts** - Standard WordPress blog posts
-- **Pages** - Static WordPress pages
-- **Custom Post Types** - Custom Post Types that support the Block Editor
-
-> **Note**: The plugin is not currently compatible with the Site Editor (Full Site Editing).
+```php
+add_post_type_support( $post_type, 'collaborative-editing' );
+```
 
 ## Requirements
 
 - **WordPress**: 6.7 or newer
-- **Gutenberg**: [Custom Gutenberg build with RTC features support](https://github.com/Automattic/gutenberg/tree/release/vip-rtc-0.1.0)
-- **WebSocket Server**: For real-time communication between users
-- **WordPress VIP**: Currently exclusive to VIP customers
+- **Gutenberg**: This plugin currently requires a custom Gutenberg build ([latest release](https://github.com/Automattic/vip-go-mu-plugins-ext/tree/trunk/vip-integrations/gutenberg))
+- **WebSocket server**: A WebSocket server facilitates real-time communication between peers. The [code is included in this repo](./websocket-server) but must be deployed separately.
 
-## Installation
+See [CONTRIBUTING.md](docs/CONTRIBUTING.md) for development setup instructions.
 
-The plugin is available to WordPress VIP customers. To enable VIP Real-Time Collaboration:
+## FAQ
 
-1. Visit the [VIP Integrations Center](https://docs.wpvip.com/integrations/center/)
-2. Enable Real-Time Collaboration with a few clicks
-3. All required infrastructure and software will be automatically configured
+### Is this plugin available to WPVIP customers?
 
-## FAQ
+VIP Real-Time Collaboration is currently in development and not yet available to all WPVIP customers.
```

Comment:
Should we also mention to get in touch with support or your TAM for more details?

Reply:
`Please open a ticket to discuss this.` would probably suffice

Reply:
Removing this question while we work on a defined path.